DTS-STN/passport-status
# Created contact us page

## [ADO-1062](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1062)

### Description

This PR adds a `Contact Us` page with links to contact the passport program and other resources as per the task created by the BA in our backlog. Currently there is no way to navigate to this page aside from manually doing so (this will come once we get more clarity on task 1054 and may result in some text on this page changing).

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/contact`
4. Ensure content and links show up properly in both EN and FR & open resource in a new tab
